api/types.proto

```diff
@@ -802,3 +811,17 @@ message RemovedNode {
 	// was issued to this node.
 	Timestamp expiry = 2;
 }
+
+message EncryptedRecord {
+	enum Algorithm {
+		option (gogoproto.goproto_enum_prefix) = false;
+		option (gogoproto.enum_customname) = "Algorithm";
+
+		NONE = 0 [(gogoproto.enumvalue_customname) = "NotEncrypted"];
+		SECRETBOX_SALSA20_POLY1305 = 1 [(gogoproto.enumvalue_customname) = "NaclSecretboxSalsa20Poly1305"];
```

Comment:
I'm not going to waste much time on this, but `Nacl` is written as `NACL` in the vendored code (I'm actually surprised it's not `NaCl`).